final-position.js

```diff
@@ -0,0 +1,24 @@
+const moves = ['north', 'north', 'west', 'west', 'north', 'east','north']  
+
+const finalPosition = function(moves){
+  let x = 0;
+  let y = 0;
+  let final = [];
+  for (let move of moves){
+  if(move === "north"){
+    y += 1;
+    } 
+  else if(move === "south"){
+    y -= 1;
+    }
+  else if(move === "west"){
+    x -= 1;
+    }
+  else if(move === "east"){
+    x += 1;
+    }
+  }
+return final = [x, y];
+}
```

Comment:
Here is a modified version of your solution that is a bit cleaner. The concept is the same, except we use an object to store the different "actions" that a move will make as anonymous functions. Then we loop through the moves array and call the function corresponding to each move on each iteration.
 
```suggestion
function finalPosition(moves) {
  let x = 0;
  let y = 0;

  const directionMap = {
    north: () => y+=1,
    south: () => y-=1,
    west: () => x-=1,
    east: () => x+=1,
  }

  moves.forEach(move => directionMap[move]());

  return [x, y];
}
```
There's some other fancy ways to do it but at a certain point you have to balance readability / simplicity with wanting to make it fancy and it's always best to keep it simple in my opinion!